syntax = "proto2";

package fueling.common.record.kinglong.proto.cybertron;

message ProtoDesc {
    required bytes desc = 1;
    repeated ProtoDesc dependencies = 2;
}
